features/promotion/managing_promotions/adding_promotion_with_rule.feature

```diff
@@ -0,0 +1,41 @@
+@managing_promotions
+Feature: Adding a new promotion with rule
+    In order to give possibility to pay less for some goods based on specific configuration
```

Comment:
Shall we then have separate features for every rule? Because in this one there are scenarios for 3 different rules implemented. One thing I'm worried is again massive number of feature files (if we want one per rule, maybe even one per action :()